examples/kebechet/alerting.md

```diff
@@ -0,0 +1,29 @@
+# Alerting
+
+Once we have all metrics collected and stored, we can define the alerts that are going to be triggered. These alerts can be provided by:
+
+- Github Issues, using Prometheus GitHub AlertManager.
+
+- email, using Prometheus webhooks.
+
+- message in Google Chat, using Prometheus webhooks and bots.
```

Comment:
add description with screenshot as an example showing how the alerts are notified via Gchat?

Reply:
I'm not sure we use this anymore actually. @harshad16 do we have some docs on this?

Reply:
Actually I think we can also remove this, as we work with github issues!